scripts/sysman/engine.yml

```diff
@@ -60,6 +60,22 @@ members:
       name: "subdeviceId"
       desc: "[out] If onSubdevice is true, this gives the ID of the sub-device"
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension properties related Engine group properties"
+version: "1.7"
+class: $sEngine
+name: $s_engine_ext_properties_t
+base: $s_base_properties_t
+members:
+    - type: uint32_t
+      name: "countOfVirtualFunctionInstance"
+      desc: "[out] Number of VirtualFunction(VF) interface associated with engine to monitor the global utilization of HW across all VF's from a Physical Function (PF) interface. These global and VF-by-VF views should provide engine group and individual engine level granularity."
```

Comment:
* (_note spaces_): "VirtualFunction(VF)" -> "Virtual Function (VF)"
* (_spell it out_): "of HW" -> "of hardware"
* (_spell it out_): "across all VF's" -> "across all Virtual Functions"

Reply:
Fixed